### PR TITLE
Update whychooseoskari.ts

### PR DESCRIPTION
**Pull request by rekjuh**
Change the text on Functionalities page, v1

### CODE REVIEW

File: utils/data/whychooseoskari.ts [hunk 1]
```diff
@@ -70,63 +70,87 @@ export const DISCOVER_FEATURE_DATA: Array<DiscoverFeatureDataType> = [
...
+          'Publish data from various different sources',
         description:
-          'Oskari allows you to connect many different data sources through a variety of interfaces. This enables a wide geospatial data service.',
+          'Add map layers using APIs (e.g. WFS, WMS, OGC API Features, WMTS)',
+      },
+      {
+        icon: '',
+        title: 'Grant rights to the map layers',
+        description:
+          'Admins set different rights for different users: control who can see map layers, edit them or add new map layers.',
```
Comment (ZakarFin) [inline utils/data/whychooseoskari.ts:87]:
```suggestion
          'Admins can set different rights for different users: control who can see map layers, edit them or add new map layers.',
```

File: utils/data/whychooseoskari.ts [hunk 2]
```diff
@@ -70,63 +70,87 @@ export const DISCOVER_FEATURE_DATA: Array<DiscoverFeatureDataType> = [
...
       },
       {
         icon: '',
-        title: 'Oskari has tools for creating and editing vector data',
+        title: 'Monitor the published map layers',
         description:
-          'Oskari has basic tools for creating vector data straight in the geoportal.',
+          'See easily which map layers are have become unavailable.',
```
Comment (ZakarFin) [inline utils/data/whychooseoskari.ts:93]:
```suggestion
          'See easily which map layers have become unavailable.',
```

File: utils/data/whychooseoskari.ts [hunk 3]
```diff
@@ -70,63 +70,87 @@ export const DISCOVER_FEATURE_DATA: Array<DiscoverFeatureDataType> = [
...
+        icon: '',
+        title: 'Create thematic maps',
         description:
-          'Oskari’s geoportal functions by itself as a tool for showing maps.',
+          'View statiscs on maps.',
       },
       {
         icon: '',
-        title: 'Publish data as embedded maps via RPC',
+        title: 'RPC enables the user to interact with embedded maps',
```
Comment (ZakarFin) [inline utils/data/whychooseoskari.ts:145]:
```suggestion
        title: 'Create interactive maps using the API for embedded maps',
```

File: utils/data/whychooseoskari.ts [hunk 4]
```diff
@@ -70,63 +70,87 @@ export const DISCOVER_FEATURE_DATA: Array<DiscoverFeatureDataType> = [
...
         description:
-          'You can publish and integrate your GIS data as an embedded map to wherever you want. The map reacts in real time to the user’s actions.',
+          'The embedded map reacts in real time to the user’s actions. Users can add new features to the map, hide/show layers, and get info from the selected features.',
```
Comment (ZakarFin) [inline utils/data/whychooseoskari.ts:147]:
```suggestion
          'The RPC API for embedded map allows developers to programmatically control the map and react to the user’s actions. This way you can you can enhance a simple embedded map to allow users to add new features to the map, hide/show layers, and get info from the selected features.',
```

Comment (rekjuh) [reply utils/data/whychooseoskari.ts:147]: 'The RPC API for embedded map allows developers to programmatically control the map and react to user’s actions. This way a simple embedded map can be enhanced to allow users to add new features to the map, hide/show layers, and get info from the selected features.',

Comment (rekjuh) [reply utils/data/whychooseoskari.ts:147]: > remove you can you can

**Pull request merged**